pages/api/users/index.ts

```diff
@@ -35,19 +35,28 @@ const userHandler: NextApiHandler = async (req: NextApiRequest, res: NextApiResp
 
     case "GET": {
       let role: Roles | undefined = undefined;
-      if (req.query && req.query.role) {
-        if (
-          ["Admin", "Teacher", "Parent", "Volunteer", "Student"].includes(req.query.role as string)
-        )
-          role = req.query.role as Roles;
-        else
-          return res
-            .status(StatusCodes.BAD_REQUEST)
-            .json("Query parameter refers to role that does not exist");
+      let onlyUnapproved = false;
```

Comment:
instead of doing it this way, we should check for 2 query parameters. approved and unapproved. So something like 
``` javscript 
// approvalStatus will be undefined is approved is not set, or it will be true/false depending on the value of approved
const approvalStatus = !req.query.approved ? undefined :  req.query.approved=='true';
if (approvalStatus) { // filter based on it }
else { //return everything }
```
Make sure to change the corresponding route in the `LeagueAPI.ts` file